dbt/adapters/databricks/python_submissions.py

```diff
@@ -201,6 +204,13 @@ def __init__(
 
     def create(self) -> str:
         # https://docs.databricks.com/dev-tools/api/1.2/index.html#create-an-execution-context
```

Comment:
isnt' it better t use https://<host>/api/2.0/clusters/get ?

Reply:
I followed the pre-existing convention in this file, which is to link the documentation for the API endpoint.

Reply:
that's now what I meant. I meant using the 2.0 API instead of the 1.2

Reply:
ahh, sorry, my bad. This API is not something you added in this PR. This is the old behavior.